2-mandatory/1-create-functions.js

```diff
@@ -33,7 +36,10 @@ Write a function that:
 - Returns a new array containing the same elements, but without the element at the passed index.
 */
 
-function remove() {
+function remove(array,index) {
+  let copyOfArray = Array.from(array);
```

Comment:
```suggestion
  let copyOfArray = arr.slice();
```
Your approach will work, and this is a good solution. Array.from creates a shallow copy and this is what was asked for. 

https://developer.mozilla.org/en-US/docs/Glossary/Shallow_copy

People would typically use slice instead. 

There is a subtle difference between slice and Array.from that is completely non obvious and not actually relevant in this case, but is interesting to know about: slice makes a sparse array and Array.from does not. Later on, come back to this and read about it. For now, take your ⭐